stdlib/Makefile

```
@@ -84,8 +84,10 @@ installopt-default:
 	  stdlib.cmxa stdlib.$(A) std_exit.$(O) *.cmx \
 	  "$(INSTALL_LIBDIR)"
 
-%-launch-info: %.info tmpheader.exe
-	@cat $^ > $@
+MANGLING = $(filter true,$(SUFFIXING))
+runtime-launch-info: tmpheader.exe
+	@{ printf '$(if $(MANGLING),$(ZINC_RUNTIME_ID_HI),\000)'; \
+     cat $^; } > $@
```

Another missing Tab.
```suggestion
	@{ printf '$(if $(MANGLING),$(ZINC_RUNTIME_ID_HI),\000)'; \
	   cat $^; } > $@
```
There’s another in 3a625a91740a6214ee4e0f25fbaab9b4d9a88935, but it vanishes post-bootstrap.